packages/core/src/common/performance/stopwatch.ts

```diff
@@ -112,6 +126,19 @@ export abstract class Stopwatch {
         return result;
     }
 
+    protected handleCompletedMeasurement(name: string, startTime: number, elapsed: number, options: LogOptions): void {
```

Comment:
Why not keep this method private or inline it? I don't see the point of overriding it in a subclass. It just adds gratuitous API surface.